cycledash/auth.py

```diff
@@ -107,3 +108,33 @@ def register():
             login_user(user)
             return redirect('/')
     return render_template('register.html', errors=errors)
+
+def check_login_from_key(api_key):
+    username, password = api_key.split(":")
+    return check_login(username, password)
+
+def load_user_from_request(request):
+    """Support for basic authorization."""
+    # first, try to login using the api_key url arg
```

Comment:
Mind dropping support for this? It's not the most secure (namely because URLs are stored in logs), and I'm not sure it buys us anything?